### ISSUE TITLE
emby running in docker permissions 

### ISSUE THREAD

**Issue by binarygeek119** [chefbennyj1/Emby.IntroSkip#3]
my emby server is running in docker on unraid 
linuxserver/emby:beta
i wast getting anywhere with running it tell i ssh in to the docker and found that fpcalc wasnt able to run
i ran
chmod +x fpcalc
then it started working

i dont know if you can fix that or not
may have contact linuxserver to fix it

just thought you should know about that

**Comment by puzzledsam** [comment]
> my emby server is running in docker on unraid
> linuxserver/emby:beta
> i wast getting anywhere with running it tell i ssh in to the docker and found that fpcalc wasnt able to run
> i ran
> chmod +x fpcalc
> then it started working
> 
> i dont know if you can fix that or not
> may have contact linuxserver to fix it
> 
> just thought you should know about that

Indeed, this issue has been there since fpcalc was included within the plugin and should probably addressed at some point. It affects everyone running the plugin on a Linux platform, including Docker. We’ll see what @chefbennyj1 can do about it.

**Comment by chefbennyj1** [comment]
It looks like we might be able to move away from fpclac, and use built in chroma-printing in ffmpeg... eventually.
The developers of emby are looking into building a version of ffmpeg which has this ability, and releasing it with the core emby application at some point in the future.
